lago/prefix.py

```diff
@@ -1225,6 +1226,7 @@ def _deploy_host(self, host):
                 return
 
             with LogTask('Wait for ssh connectivity'):
+                time.sleep(5)
```

Comment:
This will make you wait 5 seconds every time you want to deploy the host no? Maybe we can do a first try before the sleep? It seems to me also that this belongs more into the `wait_for_ssh` function, or maybe to the start one if it's only needed when booting up the vm.